diff -u qore.orig/Makefile qore/Makefile
--- qore.orig/Makefile	2014-02-05 14:20:18.000000000 -0600
+++ qore/Makefile	2014-02-05 14:17:59.000000000 -0600
@@ -1,7 +1,7 @@
-# $FreeBSD: head/lang/qore/Makefile 327741 2013-09-20 19:53:09Z bapt $
+# $FreeBSD: lang/qore/Makefile 327741 2013-09-20 19:53:09Z bapt $
 
 PORTNAME=	qore
-PORTVERSION=	0.8.7
+PORTVERSION=	0.8.8
 CATEGORIES=	lang
 MASTER_SITES=	SF/qore/qore/${PORTVERSION}/
 
diff -u qore.orig/distinfo qore/distinfo
--- qore.orig/distinfo	2014-02-05 14:20:18.000000000 -0600
+++ qore/distinfo	2014-02-05 14:17:59.000000000 -0600
@@ -1,2 +1,2 @@
-SHA256 (qore-0.8.7.tar.bz2) = 314c8c2a9394e8b366c8268183af959349b8d5eca0fcc456108a9efa71786ee7
-SIZE (qore-0.8.7.tar.bz2) = 8128234
+SHA256 (qore-0.8.8.tar.bz2) = d099bd7d14751a8d9529937116e0170b26dd431996b86bf2aa9568b661f89111
+SIZE (qore-0.8.8.tar.bz2) = 14351424
diff -u qore.orig/pkg-plist qore/pkg-plist
--- qore.orig/pkg-plist	2014-02-05 14:20:18.000000000 -0600
+++ qore/pkg-plist	2014-02-05 14:17:59.000000000 -0600
@@ -73,17 +73,11 @@
 include/qore/qore_thread.h
 include/qore/safe_dslist
 include/qore/support.h
+include/qore/QoreHttpClientObject.h
+include/qore/QoreSocketObject.h
+include/qore/qlist
 lib/libqore.la
 lib/libqore.so
-lib/libqore.so.15
-lib/qore-modules/%%PORTVERSION%%/HttpServer.qm
-lib/qore-modules/%%PORTVERSION%%/SmtpClient.qm
-lib/qore-modules/%%PORTVERSION%%/Mime.qm
-lib/qore-modules/%%PORTVERSION%%/TelnetClient.qm
-lib/qore-modules/%%PORTVERSION%%/MailMessage.qm
-lib/qore-modules/%%PORTVERSION%%/Pop3Client.qm
-lib/qore-modules/%%PORTVERSION%%/CsvUtil.qm
+lib/libqore.so.16
 libdata/pkgconfig/qore.pc
 @dirrm include/qore
-@dirrm lib/qore-modules/%%PORTVERSION%%
-@dirrm lib/qore-modules
